rotkehlchen/api/server.py

```diff
@@ -321,6 +322,7 @@
     ('/balances/historical/asset', HistoricalAssetAmountsResource),
     ('/balances/historical/asset/prices', HistoricalPricesPerAssetResource),
     ('/balances/historical/netvalue', HistoricalNetValueResource),
+    ('/blockchains/evm/transactions/refetch', RefetchEvmTransactionsResource),
```

Comment:
have it next to the other endpoints in the same path, having them disperse around the list is making ourselves the work harder